PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -32,12 +33,17 @@ var url = WScript.Arguments(0),
     project = '',
     editor = '"' + settings.disk_letter + '\\' + ( settings.x64 ? 'Program Files' : 'Program Files (x86)' ) + '\\JetBrains\\' + settings.folder_name + ( settings.x64 ? '\\bin\\phpstorm64.exe' : '\\bin\\phpstorm.exe' ) + '"';
 
+// add JSON support
+if (typeof JSON !== "object") {
+    JSON = {};
+    addJsonSupport();
+}
```

Comment:
When does this happen?

Looking at https://stackoverflow.com/questions/19445189/cscript-jscript-json I've found potentially other ways (haven't tested), that could parse a JSON string, like these:

* `eval('obj = {' + JSONstring + '}');` - poses a security risk when somebody creates a JSON file with a specific content
* including a https://github.com/douglascrockford/JSON-js file (here is how to include: https://stackoverflow.com/questions/928237/how-can-i-create-a-javascript-library-in-a-separate-file-and-include-it-in-ano)